package in.ua.denoming.juja.labs.lab38;

class TryAgainException extends Exception {

}
